performance-tools/performance-cluster/scripts/helpers/provision-gke-cluster.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
This script is a duplicate of an existing script. I don't see any changes, please re-use existing scripts.